src/types/Recipient.ts

```diff
@@ -72,6 +74,7 @@ export class NotificationUserRecipient extends BaseNotificationRecipient {
         );
 
         this.sendToAdmin = sendToAdmin;
+        this.ignorePreferences = ignorePreferences;
     }
 
     public equals(recipient: Recipient) {
```

Comment:
Could you update this equal to include the `ignorePreferences`? 

Change `return recipient.sendToAdmin === this.sendToAdmin;` to `return recipient.sendToAdmin === this.sendToAdmin && recipient.ignorePreferences === this.ignorePreferences;`